docs_src/microservices/general/index.md

```diff
@@ -131,4 +131,98 @@ Only `httpheader` is currently supported. The `headername` specifies the authent
     ```
     GET https://example.com/uom.yaml HTTP/1.1
     <name>: <contents>
-    ```
\ No newline at end of file
+    ```
+
+
+### Timestamp Precision
+
+We have several data models with a timestamp field, but the precision varies. Please refer to the following measurements and examples:
+
+- `created` and `modified` field cross different data models are in <b>milliseconds</b>
+   ```
+    {
+        "apiVersion":"v3",
+        "requestId":"479439fa-0148-11eb-adc1-0242ac120002",
+        "statusCode":200,
+        "totalCount":3,
+        "devices":[
+            {
+            "id":"55b68fcf-0fd2-445a-9fae-670b37fb9274",
+            "name":"Random-Boolean-Device",
+            "description":"Example of Device Virtual",
+            "created":1600927134931,
+            "modified":1600927134931,
+            "adminState":"UNLOCKED",
+            "operatingState":"UP",
+            "labels":[
+                "device-virtual-example"
+            ],
+            "location":"",
+            "serviceName":"device-virtual",
+            "profileName":"Random-Boolean-Device",
+            "autoEvents":[
+                {
+                "interval":"10s",
+                "onChange":false,
+                "sourceName":"Bool"
+                }
+            ]
+           }
+        ]
+    }
+    ```
+
+- `start` and `end` timestamp in the Scheduler Service are in <b>milliseconds</b>
+   ```
+   {
+    "apiVersion": "v3",
+    "statusCode": 200,
+    "totalCount": 3,
+    "intervals": [
+        {
+        "created": 1634279367311,
+        "modified": 1634279367311,
+        "id": "debade10-7838-44bd-9c09-1283813db6c8",
+        "name": "interval_4",
+        "start": "20211015T200000",
+        "end": "20211016T200000",
+        "interval": "100m"
+        }
+      ]
+    }
+    ```
+
+- `timestamp` in Metrics and System Events are in <b>nanoseconds</b>, please refer to [System Event DTO][1] for more information.
+
+- `origin` in Event and Readings is in <b>nanoseconds</b>
```

Comment:
add reference to https://docs.edgexfoundry.org/3.1/walk-through/Ch-WalkthroughReading/#origin-timestamp